packages/kbn-custom-integrations/src/components/create/form.tsx

```diff
@@ -143,7 +143,7 @@ export const CreateCustomIntegrationForm = ({
                     'customIntegrationsPackage.create.integration.name.tooltip',
                     {
                       defaultMessage:
-                        'Provide a name for the integration that will be created to organise these custom logs.',
+                        'The name of the integration that will be created to organize your custom logs.',
```

Comment:
I think we should add something about the integration being created somewhere other than in this tool tip. I would suggest something along the lines of "Elastic creates an integration to streamline connecting your log data to the Elastic Stack." to match the Syslogs onboarding. I don't know how to update this sort of thing however 😄.

Reply:
Which syslog onboarding are you referring to?

Reply:
The new system logs onboarding. Specifically this tooltip:

![image](https://github.com/elastic/kibana/assets/114418652/2600177c-7c88-48cb-be2a-633ac5d5058a)

Reply:
Found it here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_onboarding/public/components/app/system_logs/system_integration_banner.tsx#L125 Does this help?

Reply:
Sorry, what I was trying to say is that we could possibly add a line similar to that syslog tooltip ("Elastic creates an integration to streamline connecting your log data to the Elastic Stack." was my suggestion) under the **Configure integration** heading, but I don't know how to update the code to add that line if there weren't any objections to it.

![image](https://github.com/elastic/kibana/assets/114418652/cb6518c2-7d7f-4a20-9528-39e108b4eb30)

Reply:
@yngrdyn ^

Reply:
Addressed in [4e7113c](https://github.com/elastic/kibana/pull/166993/commits/4e7113c06d4995d9c442197325ab65bb6acaf0df).

<img width="930" alt="image" src="https://github.com/elastic/kibana/assets/1313018/10167080-27b0-4833-be6b-ec2ed964b200">